.github/workflows/build.yml

```diff
@@ -2,19 +2,26 @@
 # - Updating the NPM version
 # - Updating package.json in repo
 # - Building new Docker image
-# - Pushing image to registry
+# - Pushing image to Docker Hub
+# - Committing new version to repo
+# - Make GH release
 #
 # Prerequisites
-# - AKS cluster w/ attached ACR
-# - Managed identity for cluster access
-# - Github secrets: REGISTRY_USERNAME and REGISTRY_PASSWORD for ACR
+# - Github secrets:
+#   - DOCKERHUB_USERNAME and DOCKERHUB_TOKEN for publishing image to Docker Hub
+#   - AUTH_TOKEN with write permissions on repo, to push the changed package.json, as main is a protected branch
 name: Release and build new version
-on: workflow_dispatch
+on:
```

Comment:
Should only be possible on `main` branch?

Reply:
Goeie! We hebben immers geen pre-releases o.i.d.

workflow_dispatch geeft altijd de optie om de branch te kiezen. Er is (momenteel) geen mogelijkheid om dat volledig af te schermen. We zouden wel zelf een if statement toe kunnen voegen om de branch te checken:
https://github.community/t/limit-branch-for-workflow-dispatch/122099/5

Reply:
Hiet is niet fool-proof, zoals ook in de comment daarna genoemd wordt, maar biedt in ieder geval iets van bescherming. Zal het toevoegen